modules/ocf_printhost/manifests/enforcer.pp

```diff
@@ -19,7 +23,10 @@
       mode   => '0500';
 
     '/opt/share/enforcer/enforcer.conf':
-      source    => 'puppet:///private/enforcer/enforcer.conf',
+      content   => template(
+        'ocf_printhost/enforcer/enforcer.conf.erb',
+        'ocf/broker/broker.conf.erb',
+      ),
```

Comment:
Nice, this is much better, and I like how you've done the combination of the two here so you can use the broker config twice.